packages/react/src/patterns/blocks/ContentGroupPictograms/__stories__/index.scss

```diff
@@ -0,0 +1,6 @@
+@import '../../../../../../styles/scss/patterns/blocks/content-group-pictograms';
+
+.#{$prefix}--content-group-pictograms .#{$prefix}--content-group__children {
+  background: none;
+  height: auto;
```

Comment:
This shouldn't be necessary, why are we including it here?

Reply:
This was used to avoid an issue where the styling for the content group story was affecting another stories, this is fixed by now, so I removed as you said